src/deleteTask.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/no-cycle
```

Comment:
- [ ] [OPTIONAL] Please kindly avoid disabling ESlint rules to get around them. when you have ESlint errors it is usually a sign that you have an underlying problem in the way you've structured your code. Please avoid them